### DISCUSSION TITLE
Show `build.rs` in a tab like readme, versions, etc.

### DISCUSSION THREAD

**Discussion by fprasx** [rust-lang/crates.io#6094] [Feature Requests]
Showing the `build.rs` with other information like the `README.md` would be very convenient and would allow people to inspect a crate more easily to decide if they want to use it.

**Comment by Turbo87** [comment]
Thanks for the suggestion.

I think if we did that then we wouldn't restrict it to only the build.rs file but instead show all files inside the crate. Since docs.rs already provides the same functionality this does not seem like a high priority though.

**Reply by fprasx** [reply]
Ahh ok that seems reasonable. The reason I mention the `build.rs` in particular is because any code can be executed in it at compile time so some users might want to see it before doing anything (kinda like looking at a `Makefile` or `PKGBUILD`). That being said, I can see why there are bigger fish to fry.

Thank you for all the great work on crates.io!
